templatefinder/utils.py

```diff
@@ -51,14 +55,24 @@ def find_all_templates(pattern='*.html'):
         ):
             loader_class = getattr(import_module(module), klass)
             if getattr(loader_class, '_accepts_engine_in_init', False):
```

Comment:
Maybe you could rewrite that part not to use `if getattr(loader_class, '_accepts_engine_in_init', False)` - since we anyway need to handle the possible TypeError? Or is it better to keep it like that?